language-support/java/codegen/src/it/daml/Tests/EnumTest.daml

```diff
@@ -0,0 +1,7 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+daml 1.2
+module Tests.EnumTest where
+
+data EnumColor = Red | Green | Blue
```

Comment:
Please also add a record, variant, and "variantrecord" that has field of type `EnumColor`.

Reply:
This test was introduced by mistake, since damlc still  not produce `Enum` type. 
I add the  record, variant, and "variantrecord" in 

- daml-lf/encoder/src/test/lf/since_1.dev/Enum.lf
- language-support/java/codegen/src/test/java-1.dev/com/digitalasset/lf_1_dev/EnumTest.java